To: vim_dev@googlegroups.com
Subject: Patch 7.3.388
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.388
Problem:    Crash on exit when EXITFREE is defined and using tiny features.
Solution:   Check for NULL window pointer. (Dominique Pelle)
Files:	    src/buffer.c


*** ../vim-7.3.387/src/buffer.c	2011-11-30 15:40:51.000000000 +0100
--- src/buffer.c	2011-12-30 13:35:26.000000000 +0100
***************
*** 419,424 ****
--- 419,426 ----
      if (
  #ifdef FEAT_WINDOWS
  	win_valid(win) &&
+ #else
+ 	win != NULL &&
  #endif
  			  win->w_buffer == buf)
  	win->w_buffer = NULL;  /* make sure we don't use the buffer now */
*** ../vim-7.3.387/src/version.c	2011-12-30 13:09:15.000000000 +0100
--- src/version.c	2011-12-30 13:37:23.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     388,
  /**/

-- 
Be nice to your kids...  they'll be the ones choosing your nursing home.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
